src/server/server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const bodyParser = require('body-parser')
+const path = require('path');
+const app = express();
+const port = 8080;
+
+app.use(express.static(path.join(__dirname, '../..', 'build')));
+
+app.get('/ping', (_, res) => res.send('pong'));
+
+app.get('/', (_, res) => {
+  res.sendFile(path.join(__dirname, 'build', 'index.html'));
+});
+
+app.listen(port, () => {
+  console.log('Server running on http://localhost:8080/')
```

Comment:
```suggestion
  console.log(`Server running on http://localhost:${port}/`)
```
I suggest using string substitution here to dynamically display the port, as that value changes depending on where / how we deploy in the future.

Reply:
We will need to do something similar in the future when we decide to add TLS security.

Reply:
good point, changed it.